sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/Jsr305AnnotationsCheck.java

```diff
@@ -0,0 +1,1096 @@
+     */
+    protected enum NullnessAnnotation {
+
+        /** Override. */
+        OVERRIDE("Override", "java.lang"),
+        /** CheckForNull. */
+        CHECK_FOR_NULL("CheckForNull", PKG_JAVAX_ANNOTATION),
+        /** Nullable. */
+        NULLABLE("Nullable", PKG_JAVAX_ANNOTATION),
+        /** Nonnull. */
+        NONNULL("Nonnull", PKG_JAVAX_ANNOTATION),
+        /** CheckReturnValue. */
+        CHECK_RETURN_VALUE("CheckReturnValue", PKG_JAVAX_ANNOTATION),
+        /** ParametersAreNonnullByDefault. */
+        PARAMETERS_ARE_NONNULL_BY_DEFAULT("ParametersAreNonnullByDefault", PKG_JAVAX_ANNOTATION),
+        /** ParametersAreNullableByDefault. */
+        PARAMETERS_ARE_NULLABLE_BY_DEFAULT("ParametersAreNullableByDefault", PKG_JAVAX_ANNOTATION),
+        /** ReturnValuesAreNonnullByDefault. */
+        RETURN_VALUES_ARE_NONNULL_BY_DEFAULT("ReturnValuesAreNonnullByDefault",
+                "edu.umd.cs.findbugs.annotations");
+
+        /** The annotation's name. */
+        private final String annotationName;
+
+        /** The annotation's fully qualified class name. */
+        private final String fullyQualifiedClassName;
+
+        /**
+         * Constructor.
+         * @param annotationName
+         *        the annotation's name
+         * @param packageName
+         *        the package name
+         */
+        NullnessAnnotation(final String annotationName, final String packageName) {
+            this.annotationName = annotationName;
+            this.fullyQualifiedClassName = packageName + "." + annotationName;
+        }
+
+    }
+
+    /** The map of annotations against their respective names. */
+    private static final Map<String, NullnessAnnotation> STRING2ANNOTATION =
+            createString2AnnotationMap();
+
+    /** The modifiers of interest. */
+    private static final int[] DEFAULT_MODIFIERS = {
+        TokenTypes.PARAMETER_DEF,
+        TokenTypes.METHOD_DEF,
+        TokenTypes.PACKAGE_DEF,
+        TokenTypes.CTOR_DEF,
+        TokenTypes.CLASS_DEF,
+        TokenTypes.INTERFACE_DEF,
+        TokenTypes.ENUM_DEF,
+    };
+
+    /** Parameter: packages to check. */
+    private String[] packages = new String[0];
+    /** Parameter: packages to exclude from checking. */
+    private String[] excludePackages = new String[0];
+    /** Parameter: overriding return value annotations allowed. */
+    private boolean allowOverridingReturnValue;
+    /** Parameter: overriding parameter annotations allowed. */
+    private boolean allowOverridingParameter;
+
+    /** State, is a package excluded. */
+    private boolean packageExcluded;
+
+    @Override
+    public final int[] getDefaultTokens() {
+        return getAcceptableTokens();
+    }
+
+    @Override
+    public final int[] getRequiredTokens() {
+        return new int[0];
+    }
+
+    @Override
+    public final int[] getAcceptableTokens() {
+        return DEFAULT_MODIFIERS.clone();
```

Comment:
rename DEFAULT_MODIFIERS to ACCEPTABLE_MODIFIERS.

Reply:
Done.